### PR TITLE
Added comments in action-mappers.

### PR DESCRIPTION
**Pull request by TTHoang97**

### CODE REVIEW

**Review by sthoms3 — CHANGES_REQUESTED** [review]
turn into documentation comments pls

File: src/App.tsx [hunk 1]
```diff
@@ -2,11 +2,11 @@
 import React from 'react';
 import './Rms.css';
 import './App.css';
-import HomeComponent from './components/home-component/HomeContainer';
+import HomeComponent from './components/Home-component/HomeContainer';
```
Comment (jbostik) [inline src/App.tsx]: This should be lower case

File: src/App.tsx [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import EmployeeDetailsComponent from './components/employee-details-component/EmployeeDetailsContainer';
 import NavbarComponent from './components/navbar-component/NavbarComponent';
-import EmployeesComponent from './components/employees-component/EmployeesContainer';
+import EmployeesComponent from './components/Employees-component/EmployeesContainer';
```
Comment (jbostik) [inline src/App.tsx]: This should be lower case

File: src/App.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ class AppComponent extends React.Component<any, any> {
   // const login = useSelector((state: IState) => state.userState.loggedIn)
   // if loggen, we return most details for the application
   render(){
-  if (this.props.loggedIn) {
+  // if (this.props.loggedIn) {
```
Comment (jbostik) [inline src/App.tsx]: this should not be commented

File: src/App.tsx [hunk 4]
```diff
@@ -43,19 +43,19 @@ class AppComponent extends React.Component<any, any> {
...
+  //   return (
+      // <div data-test="login-content">
+      // <Router>
+      //   <Switch>
+      //     <Route path='/login' component={LoginComponent} />
+      //     <Route path='/' component={LoginComponent} />
+      //   </Switch>
+      // </Router>
+      // </div>
+    // )
```
Comment (jbostik) [inline src/App.tsx]: This should not be commented

File: src/App.tsx [hunk 5]
```diff
@@ -43,19 +43,19 @@ class AppComponent extends React.Component<any, any> {
...
   }}
-}
+// }
```
Comment (jbostik) [inline src/App.tsx]: this should not be commented

File: src/action-mappers/campus-action.ts [hunk 6]
```diff
@@ -1,3 +1,6 @@
+/**Action to change the state of campuses to all of the known campuses*/
+//Going to need a method to get only specific campuses when neccessary
+//Not sure if update method at the bottom is neccessary when useEffct/useState should run getAll again and display all of the new data.
```
Comment (jbostik) [inline src/action-mappers/campus-action.ts:3]: This should be one single documentation comment

File: src/action-mappers/employees-action.ts [hunk 7]
```diff
@@ -1,3 +1,5 @@
+/**Action to change the state of the employees to all of the known employees */
+//Not sure if update method at the bottom is neccessary when useEffct/useState should run getAll again and display all of the new data.
```
Comment (jbostik) [inline src/action-mappers/employees-action.ts:2]: These should be one single documentation comment

File: src/action-mappers/room-details-actions.ts [hunk 8]
```diff
@@ -1,3 +1,5 @@
+/**Action to change the state of room to a given room*/
+//Need to add in proper methods instead of a mock method.
```
Comment (jbostik) [inline src/action-mappers/room-details-actions.ts:2]: These should be one single documentation comment

**Review by jbostik — CHANGES_REQUESTED** [review]
Please fix the lines I've commented before pushing again. Thanks

**Review by jbostik — APPROVED** [review]

**Review by sthoms3 — APPROVED** [review]

**Review by wandeltk — APPROVED** [review]
Looks good to me!

**Pull request merged**